lib/ultrajsondec.c

```diff
@@ -438,14 +435,14 @@ static FASTCALL_ATTR JSOBJ FASTCALL_MSVC decode_string ( struct DecoderState *ds
         inputOffset ++;
         switch (*inputOffset)
         {
-          case '\\': *(escOffset++) = L'\\'; inputOffset++; continue;
-          case '\"': *(escOffset++) = L'\"'; inputOffset++; continue;
-          case '/':  *(escOffset++) = L'/';  inputOffset++; continue;
-          case 'b':  *(escOffset++) = L'\b'; inputOffset++; continue;
-          case 'f':  *(escOffset++) = L'\f'; inputOffset++; continue;
-          case 'n':  *(escOffset++) = L'\n'; inputOffset++; continue;
-          case 'r':  *(escOffset++) = L'\r'; inputOffset++; continue;
-          case 't':  *(escOffset++) = L'\t'; inputOffset++; continue;
+          case '\\': *(escOffset++) = 0x5c; inputOffset++; continue;
+          case '\"': *(escOffset++) = 0x22; inputOffset++; continue;
+          case '/':  *(escOffset++) = 0x2f; inputOffset++; continue;
+          case 'b':  *(escOffset++) = 0x08; inputOffset++; continue;
+          case 'f':  *(escOffset++) = 0x0c; inputOffset++; continue;
+          case 'n':  *(escOffset++) = 0x0a; inputOffset++; continue;
+          case 'r':  *(escOffset++) = 0x0d; inputOffset++; continue;
+          case 't':  *(escOffset++) = 0x09; inputOffset++; continue;
```

Comment:
Do we need to do this? My understanding is the C char literals are just a different way of writing integers so the former should be equivalent (and a bit more readable).

Reply:
I believe that `L'x'` literals are explicitly for `wchar_t`, so that's why I changed it. But normal `'x'` would be fine, yeah. Not sure why I didn't think of that.